assets/app/view/game/corporation.rb

```diff
@@ -638,9 +638,11 @@ def render_loans
           loan_props[:style][:color] = contrast_on(color)
         end
 
+        # the 'Loans' text is now defined in lib/engine/game/base.rb and linked here as @game.corp_loans_text.
+        # This way, games can customize the text if desired (initially used in 1849's Bonds variant)
```

Comment:
Remove the comments. They get out of date as the code evolves. If someone wants to see why the code was added, they can use blame and it'll link them back to this PR.

Reply:
done